FWCore/ParameterSet/src/types.cc

```diff
@@ -1074,24 +1202,53 @@ bool edm::encode(std::string& to, ParameterSet const& from) {
   return true;
 }  // encode from ParameterSet
 
+// ----------------------------------------------------------------------
+
+std::optional<std::string_view> edm::decode_pset_extent(std::string_view from) {
+  auto e = ParameterSet::extent(from);
+  if (e.empty()) {
+    return {};
+  }
+  return e;
+}
+
 // ----------------------------------------------------------------------
 // vPSet
 // ----------------------------------------------------------------------
-
-bool edm::decode(std::vector<ParameterSet>& to, std::string const& from) {
-  std::vector<std::string> temp;
-  if (!split(std::back_inserter(temp), from, '{', ',', '}'))
+#include <iostream>
```

Comment:
Is this `#include` a leftover from debugging?

Reply:
yep.